Thai/Travel/Travel.md

```diff
@@ -0,0 +1,7 @@
+﻿You will fall in love with Thailand. It is amazingly beautiful. We have plenty of stunning beaches and crystal clear water. Imagine you are at the sea now, what would you like to do?
+
+What do you want to do?
+
+- [Just have a chocolate while stroll on the beach](../../english/search_for_chocolate/search_for_chocolate.md)
+
+- [Have Tom Yum Kung. The food is as spicy as the weather.](../Cuisine/TomYumKung)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.